### ISSUE TITLE
How to convert the labels of Mapillary Dataset to be the same as Cityscapes?

### ISSUE THREAD

**Issue by PkuRainBow** [mapillary/inplace_abn#118]
Recently, we find that many approaches in the leaderboard of Cityscapes use the Mapillary dataset
as external datasets to achieve nearly ~1% improvement.

We are wondering how to align the label sets of Mapillary to the label sets of Cityscapes, here we have tried to choose a label-subset of Mapillary, which is consisted of only the labels existing in the Cityscapes and find that there exist a large number of void areas like below,

![image](https://user-images.githubusercontent.com/4639578/57592732-d0b4be00-756a-11e9-8b93-f0a2c657ecc5.png)

It would be great if anyone could share some advice!

**Comment by pkontschieder** [comment]
@PkuRainBow I guess most entries used Vistas for pre-training (on all its available classes) and eventually only replaced the final classification layer to accommodate for the set of Cityscapes classes during finetuning on Cityscapes data. In that way, there is no need for directly matching the classes.

**Comment by PkuRainBow** [comment]
@pkontschieder Could you share with me how much improvement do you achieve (in your approach) through using the Vistas dataset?

**Comment by AmeetR** [comment]
@pkontschieder  are you sure directly matching the classes wont' help? If you change the number of labels that Vistas is using, then it would would be optimizing for less things which is should be able to do with less parameters (such is my thought at leas), because there is at least some correlation because intrinsic dimensionality and class size. If this isn't true though, I'd like to know because it would save me  some work.

**Comment by ducksoup** [comment]
In an effort to streamline the resolution process we are closing any issue not directly related with the InPlace-ABN _library and scripts_. Please use other channels if you need to contact us about our paper or the Mapillary Vistas dataset.
